src/slices/baseApiSlice.ts

```
@@ -103,6 +103,40 @@ export const baseApiSlice = apiSlice.injectEndpoints({
 			transformResponse: handleResponse,
 		}),
 
+		contact: builder.mutation({
+			query: ({ data }: { data: any }) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)